lib/src/units_of_measurement/area.dart

```diff
@@ -470,7 +474,7 @@ class AreaRate extends UnitOfMeasurementRate<Area> {
 /// * [UnitOfMeasurementFormat]
 class AreaFormat extends _BaseAreaFormat<Area> {
   AreaFormat({
-    String pattern = '0.## ${UnitOfMeasurementFormat.valueUnitSymbolFormatSpecifier}',
+    String pattern = '0.##${UnitOfMeasurementFormat.valueUnitSymbolFormatSpecifier}',
```

Comment:
All default patterns have had the space between value and units removed because that seems far more natural and common to me. e.g. `3km` rather than `3 km`.